### PR TITLE
docs: Document settings related to UIAutomator2 mjpeg [screen] streaming feature + cleanup

### PR DESCRIPTION
**Pull request by jzaleski**
## Proposed changes

Documentation of the settings related to the UIAutomator2 mjpeg [screen] streaming feature added here: https://github.com/appium/appium-uiautomator2-server/pull/340 plus clean-up for consistency of the available options for UIAutomator2 (adding types, alphabetizing, formatting, etc.)

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

---

Travis has been slow today, please don't merge until the aforementioned PR has been merged. Thanks!

### CODE REVIEW

File: docs/en/advanced-concepts/settings.md [hunk 1]
```diff
@@ -58,19 +58,24 @@ Settings are implemented via the following API endpoints:
...
+| `enableNotificationListener` | Boolean which sets whether the Android device should enable or disable the `NotificationListener`. `true` by default.  | `false` or `true` |
+| `keyInjectionDelay` | Int (milliseconds) which is the same as [setKeyInjectionDelay](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setKeyInjectionDelay(long)). If a negative value is given, it would set to default(0 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g., `5000` |
+| `mjpegBilinearFiltering` | Int Controls whether or not to apply bilinear filtering to source screenshot resize algorithm. Enabling this flag may improve the quality of the resulting scaled bitmap, but will introduce a [small] performance hit. `false` by default | `false` or `true` |
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/settings.md:66]: What is `Int`?

Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md:66]: I'd rather remove this prefix everywhere. It does not help to understand the purpose of the setting, but rather makes more confusion

Comment (jzaleski) [reply docs/en/advanced-concepts/settings.md:66]: I think that's fine by me! Updating!

Comment (jzaleski) [reply docs/en/advanced-concepts/settings.md]: This is resolved in the latest update.

**Comment by jzaleski** [comment]
@mykola-mokhnach  this is ready for re-review -- I stripped out the types as suggested.

File: docs/en/advanced-concepts/settings.md [hunk 2]
```diff
@@ -52,25 +52,30 @@ Settings are implemented via the following API endpoints:
...
+| `enableNotificationListener` |  Controls whether the Android device should enable or disable the `NotificationListener`. Defaults to `true`.  | `false` or `true` |
+| `keyInjectionDelay` | Same as: [setKeyInjectionDelay](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setKeyInjectionDelay(long)). If a negative value is given, it would set to default(0 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g., `5000` |
+| `mjpegBilinearFiltering` | Controls whether or not to apply bilinear filtering to source screenshot resize algorithm. Enabling this flag may improve the quality of the resulting scaled bitmap, but will introduce a [small] performance hit. Defaults to `false`. | `false` or `true` |
+| `mjpegScalingFactor` | Controls the scaling factor of streamed screenshots. 100 means no scaling is applied (e.g. 100% of the original size). The bigger image size is the more CPU performance is needed to encode it. Defaults to `50`. | `1` to `100` |
```
Comment (KazuCocoa) [inline docs/en/advanced-concepts/settings.md:67]: ...encode it. Integer between `1` and 100 are available. Defaults to `50`

(Addresses available type of number)
mjpegServerFramerate is as well.

**Review by KazuCocoa — COMMENTED** [review]
thanks :)

**Review by mykola-mokhnach — APPROVED** [review]

File: docs/en/advanced-concepts/settings.md (hunk 2 above)
Comment (jzaleski) [reply docs/en/advanced-concepts/settings.md:67]: Busy morning. I will update ASAP. Thanks for the review @KazuCocoa

Comment (jzaleski) [reply docs/en/advanced-concepts/settings.md]: This is resolved w/ the latest update(s)

**Comment by jzaleski** [comment]
Thanks for the prompt reviews @mykola-mokhnach / @KazuCocoa  -- this should be all set.

**Review by KazuCocoa — APPROVED** [review]
thanks very much!

**Pull request merged**